Q3.txt

```diff
@@ -0,0 +1,46 @@
+Javascript:
+
+var b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50];
+var x=0;
+var c=[];
+var d=[];
+var z=[];
+while (x<b.length){
+	if(b[x]%2 !==0)
+	{c[x]=b[x]*2;
+	console.log(c[x]);
+	}
+	if (b[x]%2 ===0)
+	{d[x]=b[x]-1;
+	console.log(d[x]);
+	}
+	if (b[x]%5 ===0)
+	{z[x]=Math.pow(b[x], 2);
+	console.log(z[x]);
+	}
+	x=x+1;
+}
+
+Ruby:
+
+ b = [5, 10, 23, 10, 7, 9, 39, 16, 1, 12, 36, 20, 50]
+ x=0
+ c=[]
+ d=[]
+ z=[]
+while x<b.length
```

Comment:
@aqattan  try to learn to use b.each do |e| it is easier and more ruby like approach